app/controllers/jobs_controller.rb

```diff
@@ -4,7 +4,21 @@ def index
     @jobs = @company.jobs
   end
 
+  def root
+    if params[:sort] == 'location'
+      @city_jobs = Job.location
```

Comment:
perhaps a more specific method name instead of `location` would make this more clear!